### DISCUSSION TITLE
Ability to generate custom Brand ramp from one color?

### DISCUSSION THREAD

**Discussion by Aidam1** [microsoft/fluentui#24556] [Q&A] [answered]
Is there any way to generate custom Brand ramp from one primary color? I think there was a ThemeGenerator in v8 that was also used in https://www.aka.ms/themedesigner.

**Comment by micahgodbolt** [comment, marked as answer]
I've got a PR out to get the v9 theme designer into our docsite. For now, you can see the currently deployed site at https://fluentuipr.z22.web.core.windows.net/pull/24507/theme-designer/storybook/index.html?path=/story/themedesigner--page

**Reply by AkiraVoid** [reply]
There are some contrast issues after switching to dark theme. I wonder if it is designed so or will be fixed when it is released.

**Reply by mark-wiemer** [reply]
(Ref https://github.com/mark-wiemer/mark-wiemer-com/issues/22, I'm going to see if I can work on this)
